### DISCUSSION TITLE
description column

### DISCUSSION THREAD

**Discussion by ZeRenCZ** [sezanzeb/input-remapper#245] [Ideas]
it would be nice to have a 3. column so we can add a note what that macro do

https://ibb.co/VwM0rdG

**Comment by sezanzeb** [comment]
Once the macro editor arrives there will be plenty of space to add comments:

https://github.com/sezanzeb/key-mapper/issues/109#issuecomment-1001226299

**Comment by ZeRenCZ** [comment]
nice)
